tests/integration/v1/provider_charm/src/charm.py

```diff
@@ -96,9 +96,7 @@ def _relation_created(self, relation_name: str) -> bool:
             bool: True/False
         """
         try:
-            if self.model.get_relation(relation_name):
-                return True
-            return False
+            return bool(self.model.get_relation(relation_name))
```

Comment:
I wonder if there's any semantic difference between this and the following:
```suggestion
            return len(self.charm.model.relations[relation_name]) > 0
```

Reply:
Regarding using `bool`, it will return `False` if the result is an empty container, `False`, or 0,  I don't see how using `len > 0` will cause a different outcome. However, I don't know the underlying differences between the two way of fetching the relation. Do you have any guess in mind @sed-i?

Reply:
Checked. In this case seems like the two are equivalent. Resolving.

Reply:
Good catch, I've changed it.